<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionDialog</class>
 <widget class="QDialog" name="SessionDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>332</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>332</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Session Manager</string>
  </property>
  <widget class="QTabWidget" name="Sess_tabWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>14</y>
     <width>321</width>
     <height>311</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Sess_tab_Sessions">
    <property name="accessibleName">
     <string/>
    </property>
    <attribute name="title">
     <string>Session List</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>311</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="Sess_HBoxLayout_textbuttons">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="Sess_listWidget_Sessions">
        <property name="minimumSize">
         <size>
          <width>210</width>
          <height>275</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="Sess_Spacer_textbuttons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Sess_VBoxLayout_buttonsspacer">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="Sess_HBoxLayout_buttons">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="Sess_VBoxLayout_buttons">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="Sess_Spacer_load">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="Sess_pushButton_load">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>92</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>32</height>
               </size>
              </property>
              <property name="accessibleName">
               <string/>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>Load</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="Sess_Spacer_loadsave">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="Sess_pushButton_save">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>92</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>32</height>
               </size>
              </property>
              <property name="accessibleName">
               <string/>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="Sess_Spacer_savedelete">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="Sess_pushButton_delete">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>92</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>32</height>
               </size>
              </property>
              <property name="accessibleName">
               <string/>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="Sess_Spacer_delete">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="Sess_Spacer_buttonsVBoxLayout">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Sess_tab_SessionErrors">
    <attribute name="title">
     <string>Session Errors</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>1</y>
       <width>315</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="Sess_VBoxLayout_errortext">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="Sess_HBoxLayout_error">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="Sess_label_errorlist">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>Error list:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="Sess_listWidget_errorlist"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>340</y>
     <width>320</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="HBoxLayout_autocloseclose">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QCheckBox" name="Sess_checkBox_autoclose">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="text">
       <string>Auto close</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="Sess_Spacer_autocloseclose">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Sess_closeButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Sess_closeButton</tabstop>
  <tabstop>Sess_listWidget_Sessions</tabstop>
  <tabstop>Sess_pushButton_load</tabstop>
  <tabstop>Sess_pushButton_save</tabstop>
  <tabstop>Sess_pushButton_delete</tabstop>
  <tabstop>Sess_checkBox_autoclose</tabstop>
  <tabstop>Sess_tabWidget</tabstop>
  <tabstop>Sess_listWidget_errorlist</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Sess_closeButton</sender>
   <signal>clicked()</signal>
   <receiver>SessionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
